src/Chainweb/Version.hs

```diff
@@ -1033,6 +1034,13 @@ chainweb218Pact Development = (>= 500)
 chainweb218Pact (FastTimedCPM g) | g == petersonChainGraph = (> 60)
 chainweb218Pact _ = (> 24)
 
+chainweb219Pact :: ChainwebVersion -> BlockHeight -> Bool
+chainweb219Pact Mainnet01 = (>= 3_774_423) -- 2023-06-02 00:00:00+00:00
```

Comment:
My math on this gives 3,745,592:

```
# Service fork height

sample height = 3,725,432
sample date = 2023-05-15 23:52:10.123092 UTC

service date = 2023/06/01

block rate = 2,880
fork height = sample height + ((service date − sample date) × block rate)
```

Reply:
Had a bug in the specification of the dates, now I get: 3,774,392

So I can confirm your number is close enough.